test/test_mixture.py

```diff
@@ -68,6 +68,10 @@ def test_gaussian_mixture_equals_sklearn(data, model, method):
     estimator.fit(X, y)
   
     numpy.testing.assert_equal(pred_c, pred_original)
+    if model != 'EllipticEnvelope':
+        resp_original = estimator.predict_proba(X)
+        resp_c = cmodel.predict_resp(X)
```

Comment:
If this one actually gives identical output to scikit-learn predict_proba(), perhaps the method shall be named predict_proba? And then rename the existing one to predict_logproba() or similar ?

Reply:
Sure, I agree

```
predict OR resp --> predict_proba()
log_proba OR predict_proba --> predict_log_proba() // for consistency
```